### DISCUSSION TITLE
各位手搓配置的大佬看看我这个VLESS-WS 然后套CDN咋就不通呢

### DISCUSSION THREAD

**Discussion by y0un9kane** [XTLS/Xray-core#3950] [Q&A]
我只截取了入站部分，其他部分可以排除没有问题

    "inbounds": [
        {
            "listen": "0.0.0.0",
            "port": 12700,
            "protocol": "vless",
            "settings": {
                "clients": [
                    {
                        "id": "f40d11fe-b41a-9619-9713-262cfd526286",
                        "level": 0,
                        "email": "khfeomzwst"
                    }
                ],
                "decryption": "none",
                "fallbacks": [
                    {
                        "dest": 80
                    }
                ]
            },
            "streamSettings": {
                "network": "ws",
                "security": "none",
                "wsSettings": {
                    "path": "/aaa"
                }
            },
            "sniffing": {
                "enabled": true,
                "destOverride": ["http", "tls"]
            },
            "tag": "in-1"
        }
    ],
``

**Comment by lxhao61** [comment]
套 CDN 不是任何端口都支持的，修改为符合 CDN 通过的端口。
另外删除回落配置，此配置是无效的。

**Reply by y0un9kane** [reply]
谢谢大佬，我需要5-20个入站端口分别对应5-20个socks5协议出站，那表明这个VLESS-WS策略不适合我，我再找找看别的策略
